### PR TITLE
Don't generate source maps for vendor files

### PR DESCRIPTION
**Pull request by southpolesteve**
This change excludes all vendor chunks from generating source maps. In practice we rarely need these for debugging. On my machine it decreases the total build time from 8min to 2min and the dist size from 200mb to 50mb

### CODE REVIEW

**Review by vignesh-mr — APPROVED** [review]

File: webpack.config.js [hunk 1]
```diff
@@ -194,7 +204,7 @@ module.exports = function(env = {}, argv = {}) {
       filename: "[name].[chunkhash:6].js",
       path: path.resolve(__dirname, "dist")
     },
-    devtool: mode === "development" ? "cheap-eval-source-map" : "source-map",
+    devtool: false,
```
Comment (vignesh-mr) [inline webpack.config.js:207]: Looks like this is the default. Do we need this?

https://webpack.js.org/configuration/devtool/#devtool

**Review by ato9000 — APPROVED** [review]

File: webpack.config.js (hunk 1 above)
Comment (southpolesteve) [reply webpack.config.js]: Nope! Removed

**Review by languy — APPROVED** [review]

**Review by swviswan — APPROVED** [review]
:shipit:

**Comment by southpolesteve** [comment]
Unfortunately this change is causing some kind of issue preventing source maps from being properly generated. There are bug reports online about ts-loader, babel-loader, and webpack sourcemaps all not playing nice. Not going to pursue it right now.

**Pull request closed without merging**